spanner/tests/system/test_system.py

```diff
@@ -508,6 +513,86 @@ def test_transaction_read_and_insert_or_update_then_commit(self):
         rows = list(session.read(self.TABLE, self.COLUMNS, self.ALL))
         self._check_row_data(rows)
 
+    def _transaction_concurrency_helper(self, unit_of_work, pkey):
+        import threading
```

Comment:
This is standard library / this is a system test, so please just import at module scope